clients/spark/core/src/test/resources/parser-test/sst_files_generator.go

```diff
@@ -90,15 +90,30 @@ func newGenerateNanoid(size int) func() (string, error) {
 }
 
 func newGenerateFuzz() func() (string, error) {
-	return newGenerateFuzzWithSharedPrefixes(EmptyPrefix)
+	f := fuzz.NewWithSeed(FuzzerSeed)
+	return func() (string, error) {
+		var val string
+		f.Fuzz(&val)
+		return val, nil
+	}
 }
 
-func newGenerateFuzzWithSharedPrefixes(sharedPrefix string) func() (string, error) {
+func newGenerateFuzzWithSharedPrefixes() func() (string, error) {
 	f := fuzz.NewWithSeed(FuzzerSeed)
+	src := rand.NewSource(041021)
+	r := rand.New(src)
+	prefixBaseLen := 1 + r.Intn(100)
```

Comment:
Let's just take a fixed value here -- I worry that large values may behave a bit too differently (e.g. their random walk on line 115 may never go with length zero).

I'd go with something like
```suggestion
	prefixBaseLen := 1 + r.Intn(10)
```

Then line 115 goes to length 0 with probability 1/25, which is often enough.

Reply:
good point, thanks!